client/authclient.cgi

```diff
@@ -57,10 +65,10 @@ my $response = $json->decode($ua->get($call_url)->content);
 my $access_token = $response->{token};
 $call_url = $data_url . "&access_token=" . $access_token;
 $response = $ua->get($call_url)->content;
-my $cookie = CGI::Cookie->new( -name    => COOKIE_NAME,
+my $cookie = CGI::Cookie->new( -name    => AuthConfig::COOKIE_NAME,
```

Comment:
Why are AuthConfig:* not part of settings?

Reply:
I do not know.